src/widgets/current-refinements/__tests__/__snapshots__/current-refinements-test.ts.snap

```diff
@@ -102,8 +92,8 @@ exports[`currentRefinements() render() DOM output renders correctly 1`] = `
         "refinements": Array [
           Object {
             "attribute": "hierarchicalFacet",
-            "count": 6,
-            "exhaustive": true,
+            "count": null,
+            "exhaustive": null,
```

Comment:
doesn't look correct either TBH

Reply:
See https://github.com/algolia/instantsearch.js/pull/4000#discussion_r309115051.

Reply:
I fixed it for other facets than hierarchical. I'm not sure what syntax is missing for this one.

Reply:
Fixed in 132015b9b4869fdc42fe53ec03cd065142099429.